pandas/tests/indexes/multi/test_setops.py

```diff
@@ -14,9 +14,7 @@
 
 
 @pytest.mark.parametrize("case", [0.5, "xxx"])
-@pytest.mark.parametrize(
```

Comment:
Please avoid changing unrelated lines for style. We use black via precommit to autoformat the code base: https://pandas.pydata.org/pandas-docs/stable/development/contributing_codebase.html#pre-commit

Reply:
Sorry about that, ran pre-commit to address